server/order/router/order.router.js

```diff
@@ -12,25 +12,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import express from 'express';
+import Application from '../../base/application.js';
 import * as order from '../controller/order_controller';
 
-const router = express.router();
-
-router.post('/', (request, response) => {
-  if (request.body.userId) {
-    order.saveOrder(JSON.parse(JSON.stringify(request.body)));
-    response.sendStatus(200);
-  } else {
-    response.sendStatus(400);
+@Application.route('/order')
+export default class OrderRouter {
+  get(request, response) {
+    order.getOrderList(request.query.userId)
```

Comment:
I'm not sure if this is right way.
Once we authorize user connection, then we keep the session.
Okay, we should do in the past but didn't
Therefore, let's leave a TODO or FIXME comment at least.